index.html

```diff
@@ -52,13 +52,31 @@ <h1 class="header-title">
         <div class="application-demo__container">
           <img
             class="application-demo__img"
-            src="/images/Group 53 (1).png"
+            src="images/Group 53 (1).png"
             alt="application demo"
           />
           <hr class="application-demo__line" />
         </div>
       </div>
     </section>
-    <section>lcnj;sbc;aiygc;iy</section>
+    <section class="description">
+      <div class="container">
+        <h1 class="description-text">How the app works</h1>
+      </div>
+    </section>
+    <section class="description-login">
+      <div class="container">
+        <div class="header-description">
```

Comment:
Use only generic classes because you have 3 identical sections

```suggestion
        <div class="description-section__row">
```